wDeveloper/ConnectionManager.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Connection Manager
+layout: custom
+keywords: [id providers, identity providers, login, tokens, federated, enterprise]
+description: The Interject Connection Manager displays the current Web API connections associated with a company and that use a federated login.
+---
+* * *
+
+## Overview
+
+The Interject Connection Manager displays the current Web API connections associated with a company and that use a [federated login](/wAbout/Federated-Login-Design.html). When a company is [set up](/wAbout/Enterprise-Login-Setup.html) with a federated login, an Enterprise Login Code will be set up in order to direct the user to the federation's login page. This code represents a connection to the ID provider used for the federated login. All codes set up for a company will be displayed in the Connection Manager.
```

Comment:
The Interject Connection Manager displays all Web API connections that are secured with a [federated login](/wAbout/Federated-Login-Design.html). When a company is [set up](/wAbout/Enterprise-Login-Setup.html) with a federated login, an Enterprise Login Code will be associated with that login in order to direct the user to their federated login page.